_posts/2023-03-31-announcing-trino-fest-2023.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Announcing Trino Fest 2023"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+---
+
+Trino Fest 2023: Lakehouse Summer Camp
+
+Get ready to kick off your summer with Commander Bun Bun at Trino Fest 2023!
+This year's event is going virtual and will take place over two days, June 14th
+and June 15th. The focus of the event will be on Trino as a data lakehouse query
+engine, with discussions on new features and the ecosystem around Trino to
+support better data lakehouse management to keep up with data warehouses.
+
+Trino Fest 2023 is the new annual Summer event dedicated to all things Trino.
+Building on the success of last year’s [Cinco de Trino]
+({% post_url 2022-05-17-cinco-de-trino-recap %}), we’re excited to bring the
+community together once again to explore the latest trends and innovations in
+Trino and data lakehouse management. With a focus on educating, community
+collaboration, and inspiration, Trino Fest 2023 will be a valuable experience
+for anyone interested in lowering their analytics costs while still maintaining
```

Comment:
in improving their data and analytics platform.


lets get rid of the cost only angle and the warehouse angle here.